desktop-app/flow-typed/electron.js

```diff
@@ -1,6 +1,6 @@
 declare class WebviewElement extends HTMLElement {
```

Comment:
Is this intention. Also, since this is a `.js` do we need to add flow before using typings?

Reply:
I change this in order to pass `eslint` here in GitHub Action